pydm/widgets/timeplot.py

```diff
@@ -787,7 +803,33 @@ def resetUpdatesAsynchronously(self):
     updatesAsynchronously = Property("bool",
                                      getUpdatesAsynchronously,
                                      setUpdatesAsynchronously,
-                                     resetUpdatesAsynchronously)
+                                     resetUpdatesAsynchronously, designable=False)
+
+    @Property(updateMode)
+    def updateMode(self):
+        """
+        The updateMode to be used as property to set plot update mode.
+
+        Returns
+        -------
+        updateMode
+        """
+        return self._updateMode
+
+    @updateMode.setter
+    def updateMode(self, new_type):
+        """
+        The updateMode to be used as property to set plot update mode.
+
+        Parameters
+        ----------
+        new_type : updateMode
+        """
+        if new_type != self._updateMode:
+            self._updateMode = new_type
+            self.getUpdatesAsynchronously
```

Comment:
These two lines here don't do anything without the `()` to call them - I think that's probably not intentional?

Reply:
Probably the get and reset lines in this method just need to be removed

Reply:
Revised and updated